### PR TITLE
PHP improvements

### PR DESCRIPTION
**Pull request by nijel**
* Handle some missing syntax
* Improve keeping quoting style
* Add support for Laravel plurals

### CODE REVIEW

File: translate/storage/test_php.py [hunk 1]
```diff
@@ -484,57 +484,78 @@ def test_parsing_arrays_declared_in_a_single_line(self):
...
+        phpsource = """<?php
+$days = array(
+    'Sunday',
+    'Monday',
+    'Tuesday',
+    'Wednesday',
+    'Thursday',
+    'Friday',
+    "Saturday",
+);
```
Comment (unho) [inline translate/storage/test_php.py:497]: Doesn't it work if it is kept in a single line? I am asking because that is how it was found.

Comment (nijel) [reply translate/storage/test_php.py:497]: Parsing works just fine, but serializing formats it this way, that's why I've changed it.

Comment (unho) [reply translate/storage/test_php.py:497]: Ok, I guess we will have to allow this to happen. On the other PHP code should be like this, so I guess it is not that bad.

**Comment by unho** [comment]
@nijel lgtm

**Pull request merged**